src/components/augmint-ui/button/styles.js

```diff
@@ -13,13 +13,18 @@ const BaseButton = `
     cursor: pointer;
     text-transform: uppercase;
     font-weight: 200;
-    margin: 0 .25em 0 0;
+    margin: 0 .95em 0 0;
     padding: 0 14px;
     border-radius: 4px;
     height: 42px;
     line-height: 42px;
     font-size: 13px;
     letter-spacing: normal;
+    border: 1px solid rgba(0,0,0,.15);
```

Comment:
we have a few predefined colour variables, could you either use one of those or add to to the list pls?

Reply:
done.
(btw i actually copied it from line 71 @ `augmint-ui/button` :D )